libcxx/docs/ReleaseNotes/22.rst

```diff
@@ -102,6 +102,8 @@ Improvements and New Features
 Deprecations and Removals
 -------------------------
 
+- Non-conforming extension ``launch::any`` is deprecated. It will be removed in LLVM 23.
```

Comment:
Adding names continues to be conforming. Extensions are a thing. Also, this should be removed in LLVM 24 at the earliest.

Reply:
I'm not sure whether this is conforming. At least, it seems to me that we haven't completed a C++11 paper ([N3188](https://wg21.link/n3188)).